tests/cql/CqlListOperatorsTest.xml

```diff
@@ -20,11 +20,11 @@
 		</test>
 		<test name="SortDatesAsc">
 			<expression>({ DateTime(2012, 10, 5, 10), DateTime(2012, 1, 1), DateTime(2012, 1, 1, 12), DateTime(2012, 10, 5) }) S sort asc</expression>
-			<output>{DateTime(2012, 1, 1), DateTime(2012, 1, 1, 12), DateTime(2012, 10, 5), DateTime(2012, 10, 5, 10)}</output>
+			<output>{ @2012-01-01T, @2012-01-01T12, @2012-10-05T, @2012-10-05T10 }</output>
 		</test>
 		<test name="SortDatesDesc">
 			<expression>({ DateTime(2012, 10, 5, 10), DateTime(2012, 1, 1), DateTime(2012, 1, 1, 12), DateTime(2012, 10, 5) }) S sort desc</expression>
-			<output>{DateTime(2012, 10, 5, 10), DateTime(2012, 10, 5), DateTime(2012, 1, 1, 12), DateTime(2012, 1, 1)}</output>
+			<output>{ @2012-10-05T10, @2012-10-05T, @2012-01-01T12, @2012-01-01 }</output>
```

Comment:
Should the 4th value have a `T` on the end?

Reply:
Absolutely, fixed